docker-compose.yml

```diff
@@ -6,7 +6,6 @@ services:
       context: .
       dockerfile: Dockerfile-app
     restart: always
-    image: "iit-backend/app"
```

Comment:
we don't want to actually push this change so don't forget to revert this

Reply:
@123chrisc

Reply:
Sorry for late follow-up on this. Did not open GitHub for a hot minute. Is this change sorted or do I need to go back and reverse it?

Reply:
@123chrisc you need to undo this change, i.e. put this line back please.